definitions/contested/json/ComplexTypes/ComplexTypes.json

```diff
@@ -1270,6 +1270,22 @@
     "ElementLabel": "File name",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "DocumentDetails",
+    "ListElementCode": "DocumentFileName",
+    "FieldType": "Text",
+    "ElementLabel": "Name",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "DocumentDetails",
+    "ListElementCode": "DocumentType",
```

Comment:
why is this a Text and not a fixed list? what kind of document types are accepted? is it document file extension instead?